### PR TITLE
DX testing round improvements - ADOPT-1570

### PR DESCRIPTION
**Pull request by JustinaPetr**
Addressing a few DX improvement points identified during the DX testing sessions:

- Making the Wheel installation more explicit [linear issue](https://linear.app/3boxlabs/issue/ADOPT-1679/make-the-wheel-installation-more-explicit-on-the-docs)

- Making it easier to find the daemon.config.json file [linear issue](https://linear.app/3boxlabs/issue/ADOPT-1570/confusing-to-find-the-path-of-ceramicdaemonconfigjson-for-the-new-devs)

### CODE REVIEW

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx [hunk 1]
```diff
@@ -3,7 +3,19 @@ import TabItem from '@theme/TabItem'
...
+The first step to build with ComposeDB is setting up your development environment. Currently, there are two ways of how you can configure your working environment:
+- [Using the Wheel](#installation-using-wheel) - the recommended and the easiest way to configure your working environment and install the necessary dependencies.
+- [Using JavaScript package managers](#installation-using-javascript-package-managers) - an alternative, more manual, way to configure your working environment which supports `npm`, `pnpm` and `yarn`.
+
+:::note
+
+**Supported operating systems:**
+- Linux
+- Mac 
+- Windows (**only WSL2**)
```
Comment (velvet-shark) [inline website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:15]: Link "WSL 2" to https://learn.microsoft.com/en-us/windows/wsl/install

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx [hunk 2]
```diff
@@ -417,11 +423,32 @@ Indexing is one of the key features of ComposeDB. In order to notify the Ceramic
...
+
+Copy this authenticated DID key and store it in a secure place, just like with your private key above. This DID key will have to be provided in your Ceramic node’s configuration file which will ensure that only authorized users can make changes to your application, e.g. deploy models on your Ceramic node.
+
+:::
 
 ### Using your account
 
-Provide the authenticated DID by opening the node configuration file which defaults to `~/.ceramic/daemon.config.json` and specifying it in the `admin-dids` section of the file as shown below:
+The very first time you spin up a Ceramic node, a node configuration file is automatically created for you where you can configure how your node is operated. Here you have to provide the DID key which is authorised to interact with the Admin API.
+The Ceramic node configuration file will be created inside of the automatically created directory `./ceramic` which can be access using the following command:
```
Comment (velvet-shark) [inline website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:441]: While "~/.ceramic" should be clear that it's in the user's home directory, nobody knew this during user testing.

So maybe let's make it even more explicit?

---

The Ceramic node configuration file will be created inside of the automatically created directory `./ceramic` in your home directory (usually `/home/USERNAME/` on Linux or `/Users/USERNAME/` on Mac)

This directory can be accessed using the following command:

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx [hunk 3]
```diff
@@ -417,11 +423,32 @@ Indexing is one of the key features of ComposeDB. In order to notify the Ceramic
...
+
+```bash
+cd ~/.ceramic
+```
+
+Inside of this directory you should see the following files:
+- `daemon.config.json` - your Ceramic node configuration file
+- `statestore` - a local directory for [persisting the data](https://developers.ceramic.network/run/nodes/nodes/#ceramic-state-store) 
+
+Apen the `daemon.config.json` file using your preferred code editor and provide the authenticated DID, generated in the [generate your account](#generate-your-account) step of this guide, in the `admin-dids` section of the file as shown below:
```
Comment (velvet-shark) [inline website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:451]: Typo: "Open"

**Review by velvet-shark — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx (hunk 3 above)
Comment (JustinaPetr) [reply website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:451]: Great catch! Fixed.

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx (hunk 2 above)
Comment (JustinaPetr) [reply website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:441]: Great suggestion! Added.

**Review by velvet-shark — APPROVED** [review]

**Pull request merged**